project/npda/general_functions/retrieve_pdu.py

```diff
@@ -33,7 +33,7 @@ def retrieve_pdu_list():
     """
 
     url = settings.RCPCH_NHS_ORGANISATIONS_API_URL
-    request_url = f"{url}/paediatric_diabetes_units/extended"
+    request_url = f"{url}/paediatric_diabetes_units/extended/"
```

Comment:
Have entered this url (with the preceding url from example.env) into my browser and receiving a 500 error. Is the API definitely live? Or have I missed something? Should the API not respond with a JSON of the PDUs?